base/src/test/java/uk/ac/ebi/atlas/search/suggester/SolrSuggestionReactSelectAdapterTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.ac.ebi.atlas.search.suggester;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import uk.ac.ebi.atlas.solr.BioentityPropertyName;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.RandomStringUtils.randomAlphanumeric;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.ac.ebi.atlas.solr.cloud.collections.BioentitiesCollectionProxy.ID_PROPERTY_NAMES;
+
+class SolrSuggestionReactSelectAdapterTest {
+    private static final Gson GSON = new Gson();
```

Comment:
Can we not use the  `GsonProvider` class here?

Reply:
Since it’s a test, I think it’s more correct to use a different instance of `Gson` not to mix responsibilities.